Session03/Session03Exersise01/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Session03Exersise01
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double doubleMid = 0;
+            double dblSum = 0;
+
+            Console.WriteLine("Ange ett antal siffror, separerat med kommatecken");
+            var input = Console.ReadLine();
+            var inputArray = input.Split(",", StringSplitOptions.RemoveEmptyEntries);
+            double?[] numberArray = new double?[inputArray.Length];
+
+            for (int i = 0; i < inputArray.Length; i++)
+            {
+                bool parsed;
+
+                try
+                {
+                    numberArray[i] = Convert.ToDouble(inputArray[i]);
+                    parsed = true;
+                }
+
+                catch (Exception)
+                {
+                    numberArray[i] = 0;
+                    parsed = false;
+                }
+
+                if (parsed == false)
+                {
+                    continue;
+                }
+
+                dblSum += numberArray[i].Value;
+            }
+            doubleMid = dblSum / numberArray.Length;
+
+
+            Console.WriteLine($"Min value is: {numberArray.Min()} Maxvalue is: {numberArray.Max()} Total value is: {dblSum} Midvalue is: {doubleMid}");
```

Comment:
Kul att du löste uppgiften! 
Notera att snittvärdet i denna implementation blir fel om ett av värdena inte parseas korrekt iomed att divisionen baseras på `numberArray.Length` och inte en egen variabel likt `int parsedNumberCount`.

Reply:
Ah! makes sence. En bättre lösning skulle alltså vara en int som har koll på korrekt parsning och använda den för att få fram rätt värde att dividera med? Vilket var det du skrev.

Reply:
Ja precis!

Reply:
Jag har ändrat så att jag har en int nu som räknar antal lyckade parsingar. Det gör att det funkar med beräkningen av medelvärdet. Det uppstår dock ett problem om man skriver ett felaktigt värde. Felet som blir är att den sätter minvärdet till 0? Blir det så för att den i arrayen tilldelas värde noll?
Alltså om jag till exempel skriver in i konsollen: 3,4,r   så lyckas parsing med 3 och 4 och beräkningen utförs på det som lyckas. R'et däremot används ju inte men jag vill få för mig att i arrayen så upptas 3 positioner varav 2 är lyckade och 1 är misslyckad. Eftersom att jag använder numberArray.Min för att få fram det lägsta inmatade värdet så tar den r'et som blivit en nolla och visat lägsta värdet 0 istället för 3. 
Hur löser jag det?
Du behöver inte svara idag söndag då det är ledig dag ;)